book/website/reproducible-research/code-quality/code-quality-style.md

```diff
@@ -2,24 +2,24 @@
 
 A coding style is a set of conventions on how to format code.
 For instance, what do you call your variables? Do you use spaces or tabs for indentation? Where do you put comments?
-Consistently using the same style throughout your code, makes it easier to read.
-Code that is easy to read, is easier to understand for yourself as well as potential collaborators.
-Therefore adhering to a coding style reduces the risk of mistakes and makes it easier to work together on software.
-[Improving software quality, why Coding Style Matters](http://coding.smashingmagazine.com/2012/10/25/why-coding-style-matters/) is a nice article on why coding styles matter and how they increase software quality.
+Consistently using the same style throughout your code makes it easier to read.
+Code that is easy to read is easier to understand by you as well as by potential collaborators.
+Therefore, adhering to a coding style reduces the risk of mistakes and makes it easier to work together on software.
+[Why Coding Style Matters](http://coding.smashingmagazine.com/2012/10/25/why-coding-style-matters/) is a nice article on why coding styles matter and how they increase software quality.
 
 For example, [PEP8](https://www.python.org/dev/peps/pep-0008/) is the most widely used Python coding style and [ECMAScript 6](http://es6-features.org/) aka [ES6](http://es6-features.org/) is the scripting-language specification standardized by ECMA International for programming in Javascript.
 
 For commonly used style guides for various programming languages see the [Language Guides](https://guide.esciencecenter.nl/best_practices/language_guides/languages_overview.html).
-Google also has a style guide for many languages [google style guide page](https://code.google.com/p/google-styleguide/) that are used in open source projects originating out of Google.
+Google also has a [style guide](https://code.google.com/p/google-styleguide/) for many languages that are used in open source projects originating out of Google.
```

Comment:
I thought this looked better with the hyperlink included in the sentence itself.